lib/metadata/testSymbol.cpp

```diff
@@ -0,0 +1,37 @@
+#include <Symbol.hpp>
+#include <cassert>
+#include <iostream>
+#include <set>
+#include <unordered_set>
+
+using namespace chord;
+
+int main() {
```

Comment:
Same here.

Reply:
It seems one of my comments got lost.   This was to say these main() functions for tests should be converted to `boost::test`, see `tests/boost/*` as examples.   This way the tests will run as part of the CI process.

Reply:
Okay!